.travis.yml

```diff
@@ -2,24 +2,42 @@ addons:
   firefox: latest-esr
 language: python
 sudo: false
-python:
-  - "2.7"
 install:
-  #- pip install pep8 --use-mirrors
-  #- pip install pyflakes --use-mirrors
-  - pip install robotframework
-  - pip install selenium==2.53.6
-  - pip install decorator
-  - pip install mockito
-  #- "if [[ ${BROWSER} == 'ie' ]]; then pip install ie_only_pkg; fi"
+  - pip install .
+  - pip install -r requirements.txt
+  - pip install -r requirements-dev.txt
 before_script:
   - "export DISPLAY=:99.0"
   - "sh -e /etc/init.d/xvfb start"
 script:
-  - "firefox --version"
-  - "python test/run_tests.py python $BROWSER --noncritical known_issue_-_travisci"
+  - "python test/run_tests.py python \"$BROWSER\""
 env:
-  matrix:
-  - BROWSER=firefox
-  #- BROWSER=ie # will be added when we have completed SauceLabs CI
-  #- BROWSER=safari # will be added after #247 merged
+  global:
+    - secure: "QyY0j8C+hVhsGLv4YU0UMvLm5wBDg6xT5J0Ee77C/JAQ6vcdCMpTfWI8oDxTZb7pBhpDkYuNe747yCYF5PrdBYQbO08FilxctReuJfMXcH44rZC5q49mUmPamkvg1LkzIWuC1s9uKnjlvyDT5ojuPXZl2Vn0gIhSIg+yOyBkM1U="
+    - secure: "HJaJTYauaBZODMDkbnRVsiCk+t2uVtztkkJ5V4dXSnE1uIBms2PVbufQwb+YNvzHkGheLtpnt/hLM1HhmSFgUtOCf+eumQ6kaLR2KRVEysXwWJzgZoLBidNBOy5rvAIsc9JJbDaIgYVGzyaJGTP9AMb8CyvIPE//lH10sf+GB7g="
+matrix:
```

Comment:
No it is not yet compatible. But the aim for the 2.0 release, which hopefully is the next release, is to make it compatible with Python 3.